mlflow/environment_variables.py

```diff
@@ -0,0 +1,44 @@
+"""
+This module defines environment variable names used in MLflow.
+"""
+import os
+
+
+class _EnvironmentVariable:
+    """
+    Define a environment variable
+    """
+    def __init__(self, name, type, default):
+        self.name = name
+        self.type = type
+        self.default = default
+
+    def get(self):
+        """
+        Get environment variable value.
+        """
+        val = os.getenv(self.name)
+        if val:
+            try:
+                return self.type(val)
+            except Exception as e:
+                raise ValueError(f"Failed to convert {val} to {self.type} for {self.name}: {e}")
+        return self.default
+
+    def __str__(self):
+        return f"Environment variable: name={self.name}, type={self.type}, default={self.default}"
```

Comment:
```suggestion
        return f"EnvironmentVariable(name={self.name}, type={self.type}, default={self.default})"
```

Can we use a format similar to `namedtuple`?